src/frontend-build.sh

```diff
@@ -83,6 +117,9 @@ docker run -i --name $CONTAINER_NAME \
   -e SERVER_NAME=$SERVER_NAME \
   -e INCLUDE_CHROME_CONFIG \
   -e CHROME_CONFIG_BRANCH \
+  -e APP_NAME=$APP_NAME \
```

Comment:
Flexibility to specify APP_NAME, SERVER_NAME and DIST_DIR from parent pr_check.sh file.
related PR: https://gitlab.cee.redhat.com/insights-platform/frontend-build-container/-/merge_requests/24